agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/DatabaseClientTracer.java

```diff
@@ -151,8 +151,7 @@ protected void onPeerConnection(final Span span, final InetSocketAddress remoteC
 
   protected void onPeerConnection(final Span span, final InetAddress remoteAddress) {
     if (remoteAddress != null) {
-      span.setAttribute(SemanticAttributes.NET_PEER_NAME.key(), remoteAddress.getHostName());
-      span.setAttribute(SemanticAttributes.NET_PEER_IP.key(), remoteAddress.getHostAddress());
+      BaseDecorator.setPeer(span, remoteAddress.getHostName(), remoteAddress.getHostAddress());
```

Comment:
Please create `BaseTracer`, define `setPeer` method there and then delegate to `BaseDecorator` as here. Then we will have to change only one place when in the future we replace all decorators with tracers